nncf/quantization/algorithms/weight_compression/openvino_backend.py

```diff
@@ -162,7 +161,7 @@ def transform_model(
                         should_add_convert_node = True
                         break
 
-            weight = Tensor(get_const_value(const_node, np.float32 if const_dtype == ov.Type.bf16 else None))
+            weight = Tensor(get_const_value(const_node, const_dtype))
```

Comment:
If I'm not mistaken, then you removed the bf16 to fp32 conversion.

```suggestion
            weight = Tensor(get_const_value(const_node))
```

Reply:
Done.